packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -103,10 +103,8 @@ function PreviewView() {
   return (
     <div className="panel-view">
       <div className="button-group-top">
-        <UrlBar project={project} disabled={devicesNotFound} />
-
+        <UrlBar key={resetKey} project={project} disabled={devicesNotFound} />
```

Comment:
why do we pas project as a prop here? Is UrlBar Ever used outside of the ProjectContextProvider?

Reply:
I've changed it to use useProject instead of props.